src/test/test.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+   Copyright The Overlaybd Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#include <gtest/gtest.h>
+#include "photon/common/alog.h"
+#include "photon/thread/thread.h"
+#include "photon/net/http/server.h"
+#include "photon/net/socket.h"
+#include "photon/photon.h"
+#include "photon/net/http/url.h"
+#include "photon/net/socket.h"
+#include "photon/io/fd-events.h"
+#include "photon/net/curl.h"
+
+#include <fcntl.h>
+
+using namespace photon::net;
+using namespace photon;
+
+photon::net::EndPoint parse_endpoint(std::string_view str) {
+    photon::net::http::URL url(str);
+    std::string host = url.host().data();
+    auto pos = host.find(":");
+    if (pos != host.npos) {
+        host.resize(pos);
+    }
+    return {photon::net::IPAddr(host.c_str()), url.port()};
+}
+
+
+TEST(ImageTest, AccelerateURL) {
+    const char *addr = "https://localhost:65001/accelerate";
```

Comment:
It doesn't make any sense for this test...
Does the function 'check_accelerate_url(std::string_view a_url)' needs to be tested?

Reply:
Unittest updated.